hub/core/tensor.py

```diff
@@ -105,11 +105,10 @@ def append_tensor(
         TensorUnsupportedSampleType: If type of the sample is not supportes.
     """
 
-    if isinstance(sample, (np.ndarray, int, float)):
+    if isinstance(sample, (np.ndarray, int, float, list)):
```

Comment:
Fixed a related bug. The `else` here is meant for the `hub.load` specialization, and shouldn't be triggered by `tensor.append([1,2,3])`